motion_control.c

```diff
@@ -196,7 +196,7 @@ void mc_arc(float *position, float *target, float *offset, float radius, float f
 // Execute dwell in seconds.
 void mc_dwell(float seconds) 
 {
-  report_status_message(STATUS_OK); //report that we are dwelling
+  /* TODO: Report a dwell status? -JC */
```

Comment:
The intent was to only report an 'OK' when we received text.  Once we start executing, the only thing we report are limits/status/errors.